tpl/tplimpl/template.go

```diff
@@ -112,15 +112,25 @@ func (t *templateHandler) Lookup(name string) *tpl.TemplateAdapter {
 		// in the text template collection.
 		// The templates are stored without the prefix identificator.
 		name = strings.TrimPrefix(name, textTmplNamePrefix)
-		return t.text.Lookup(name)
+
+		te := t.text.Lookup(name)
+		if te != nil {
+			te.Metrics = t.Deps.Metrics
```

Comment:
You should set the `Metrics` at template creation time.

Reply:
I think you're going to have to help me out here.

Reply:
OK.